src/rtp/ntp.c

```diff
@@ -64,15 +79,26 @@ int ntp_time_get(struct ntp_time *ntp)
 	GetSystemTimeAsFileTime(&now.ft);
 	tv.tv_usec = (long) ((now.ns100 / 10LL) % 1000000LL);
 	tv.tv_sec = (long) ((now.ns100 - 116444736000000000LL) / 10000000LL);
+
+	*jfs = now.ns100 / (10 * 1000);
 #else
 	if (gettimeofday(&tv, NULL) != 0)
 		return errno;
+
+	*jfs  = (uint64_t)tv.tv_sec * (uint64_t)1000;
+	*jfs += tv.tv_usec / 1000;
```

Comment:
we now have a portable `tmr_jiffies_usec` which returns the time in microseconds.

should we consider using it here ?

Reply:
I didn't know jiffies switched to MONOTONIC clock. This completely breaks synchronization with NTP. So (a) no, switching to `tmr_jiffies_usec` or `tmr_jiffies` is not a valid replacement and (b) using `tmr_jiffies` in `rtcp_sess_rx_rtp` is no longer valid. `txstat::jfs_ref` must be obtained using REALTIME clock. And should no longer be called "jiffies" for that matter since it uses a different clock.